waiter/src/waiter/state/responder.clj

```diff
@@ -470,10 +471,10 @@
           response-code (if instance-not-allowed? :in-use :blacklisted)]
       {:current-state' (if (= :blacklisted response-code)
                          (-> current-state
-                           ; mark instance as blacklisted and set the expiry time
-                           (update-in [:instance-id->state instance-id] sanitize-instance-state)
-                           (update-in [:instance-id->state instance-id] update-status-tag-fn #(conj % :blacklisted))
-                           (update-state-by-blacklisting-instance-fn cid instance-id blacklist-period-ms))
+                             ; mark instance as blacklisted and set the expiry time
+                             (update-in [:instance-id->state instance-id] sanitize-instance-state)
+                             (update-in [:instance-id->state instance-id] update-status-tag-fn #(conj % :blacklisted))
+                             (update-state-by-blacklisting-instance-fn cid instance-id blacklist-period-ms))
```

Comment:
Shams and I have been debating what our standard indention style should be for threading macros. For now, I think's its best to not change it if you're not actually editing. Please un-indent these to revert of the whitespace change.